ckim105/SER316-Group-2
# Us#9 update color schemes

To test: 

1. Run ScrumBox with Gradle run
2. Work Panel and Daily Items Panel should have updated color schemes
3. Update color schemes for borders and left menu icons

 > Developer Checklist
